### DISCUSSION TITLE
Check out that repo and maybe leave a ⭐

### DISCUSSION THREAD

**Discussion by LeleDallas** [Schweinepriester/achievement-testarea#42] [Ideas]
Hey guys!
I wanted to share with you a project I've been working on that I'm really excited about. 
It's a digital version of the popular game `Cards Against Humanity`.

You can find the GitHub repository for the game here: [Cards Against Humanity Game](https://github.com/LeleDallas/Cards-Against-Humanity)
If you have a moment, I'd be thrilled if you could take a look at the repository and leave a star ⭐️.

Thank you so much for your time, and I hope you have a blast playing the game!

**Comment by ternera** [comment]
Please star and I will return it: https://github.com/ternera/ternera.org

**Comment by AdityaNow** [comment]
Done 😃 (feel free to star any of my repo https://github.com/AdityaNow/NavtorAssignment & https://github.com/AdityaNow/Wikipedia)
Help me get the badge :star:
Thank you

**Comment by ternera** [comment]
Please star my repository and I will return it: https://github.com/ternera/hacker101-ctf

**Comment by ganochenkodg** [comment]
@LeleDallas done, and my pls https://github.com/ganochenkodg/qdrant-operator

**Reply by AdityaNow** [reply]
Done 😃 please star my repo [https://github.com/AdityaNow/NavtorAssignment](https://github.com/AdityaNow/NavtorAssignment)
Help me get the badge ⭐
Thank you

**Reply by ganochenkodg** [reply]
did it already :)

**Reply by AdityaNow** [reply]
Thank you sir!

**Reply by AdityaNow** [reply]
>

done [NCherfaoui](https://github.com/NCherfaoui)
  :)
